doc/quickstart.md

```diff
@@ -0,0 +1,174 @@
+
+Firstly, clone this GitHub repository. 
+
+```
+git clone https://github.com/intel/multus-cni.git && cd multus-cni
+```
+
+We'll apply files to `kubectl` from this repo. The files we're applying here specify a "Daemonset" (pods that run on each node in the cluster), this Daemonset handles installing the Multus CNI binary, dropping a default configuration on each node in the cluster -- and then also installs Flannel to use as a default network.
+
+```
+$ cat ./images/{multus-daemonset.yml,flannel-daemonset.yml} | kubectl apply -f -
+```
+
+### Validating your installation
+
+Generally, the first step in validating your installation is to look at the `STATUS` field of your nodes, you can check it out by looking at:
+
+```
+$ kubectl get nodes
+```
+
+This will show each of the nodes in your cluster, take a look at the `STATUS` field, and look for `Ready` to appear for each of your nodes. This readiness is determined by the presence of a CNI configuration file on each of the nodes, and when that file appears.
+
+You may also wish to start any pod in your cluster (without any further configuration), and validate that it works as you'd otherwise expect -- especially that it can communicate over the default network.
+
+## Creating additional interfaces
+
+The first thing we'll do is create configurations for each of the additional interfaces that we attach to pods. We'll do this by creating Custom Resources. Part of the quickstart installation creates a "CRD" -- a custom resource definition that is the home where we keep these custom resources -- we'll store our configurations for each interface in these.
+
+### CNI Configurations
+
+Each configuration we'll add is a CNI configuration. If you're not familiar with them, let's break them down quickly. Here's an example CNI configuration:
+
+```
+{
+  "cniVersion": "0.3.0",
+  "type": "loopback",
+  "additional": "information"
+}
+```
+
+CNI configurations are JSON, and we have a structure here that has a few things we're interested in:
+
+1. `cniVersion`: Tells each CNI plugin which version is being used and can give the plugin information if it's using a too late (or too early) version.
+2. `type`: This tells CNI which binary to call on disk. Each CNI plugin is a binary that's called. Typically, these binaries are stored in `/opt/cni/bin` on each node, and CNI executes this binary. In this case we've specified the `loopback` binary (which create a loopback-type network interface).
```

Comment:
Just notes that might be helpful to add, but don't feel you have to:
* Does my CNI need to be in place (/opt/cni/bin/) before the quick start is run for it to be propagated to each node?
* Along the same lines, if I'm debugging and update the CNI, do I need to manually propagate it to each node?

Reply:
Cool, made a note to check the contents of the directory to make sure your binaries are on disk. And then also made a note about how the configs are read on pod creation & deletion.